about.html

```diff
@@ -13,59 +13,79 @@
 
 <body>
     <div class="container about-page">
+     <header>
+        <nav>
+          <img src="img/lambda-black.png" alt="Lambda">
+          <a href="index.html">Home</a>
+          <a href="about.html">About</a>
+          <a href="#">Products</a>
+          <a href="#">Blog</a>
+          <a href="form.html">Contact</a>
```

Comment:
Tried clicking on the Contact and I see the file but it's currently empty in Github. Perhaps you didn't push it up after you saved it on your computer?